src/link/persisted-queries/index.ts

```diff
@@ -175,7 +157,7 @@ export const createPersistedQueryLink = (
       let retried = false;
       let originalFetchOptions: any;
       let setFetchOptions = false;
-      const retry = (
+      const handlePossibleRetry = (
```

Comment:
```suggestion
      const maybeRetry = (
```

I tend to like the prefix "maybe" for stuff like this to indicate conditionality. I'm not stuck on this though so pick whichever name you like more.

Reply:
I don't have a strong opinion here, changing it :)